File: deaf_grandma.js
function deafGrandma() {
  let count = 0;
  console.log("HEY KID!");
  while (count < 2) {
    let userInput = prompt("Enter message");
    let capital = userInput.toUpperCase();
    if (userInput === "") {
      console.log(`WHAT?!`);
    } else if (capital !== userInput) {
      console.log(`SPEAK UP, KID!`);
    } else if (userInput === "GOODBYE!") {
      count++;
      if (count < 2) console.log(`LEAVING SO SOON?`);
    } else if (capital === userInput) {
      console.log(`NO, NOT SINCE 1946!`);
    }
  }
  console.log(`LATER, SKATER!`);
}

deafGrandma();

File: deaf_grandma.py
def deaf_grandma():
    count = 0
    print('HEY KID!')
    while count < 2:
        userInput = input("Enter message\n")
        capital = userInput.upper()
        if userInput == "":
            print('WHAT?!')
        elif capital != userInput:
            print('SPEAK UP, KID')
        elif userInput == 'GOODBYE!':
            count += 1
            if count < 2:
                print('LEAVING SO SOON')
        elif capital == userInput:
            print('NO, NOT SINCE 1946!')
    print('LATER, SKATER!')


deaf_grandma()
